index.html

```diff
@@ -1465,6 +1465,37 @@ <h3>Credential Subject</h3>
 }
         </pre>
 
+        <p>
+It is possible to express information related to multiple <a>subjects</a> in a
+<a>verifiable credential</a>. The example below specifies two subjects,
+the first is a recipient of a Bachelor Degree, the second is the university
```

Comment:
a little punctuation tweak here.  Also, note that `bachelor's degree` is the generic, compared to `Bachelor of Science Degree in Biology`, `Bachelor of Arts Degree`, or similar for the specific.
```
specifies two subjects,
the first is a recipient of a bachelor's degree, the second
```
-> 
```
specifies two subjects;
the first is a recipient of a bachelor's degree, and the second
```
or -> 
```
specifies two subjects:
the first is a recipient of a bachelor's degree; the second
```

Reply:
@TallTed I'm going to simplify this example to a spousal relationship. Were I not to do that, I would have pulled your suggestion in.